apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -36,7 +36,8 @@ public function execute($request)
             && arStorageServiceUtils::getAipDownloadEnabled()
         );
 
-        $this->masterCopy = $this->resource->getRepresentationByUsage(QubitTerm::MASTER_ID);
+        $id = QubitTerm::MASTER_ID;
```

Comment:
This line is unnecessary, the `$id` variable is never used.

Reply:
@djjuhasz it is used in line 40

Reply:
No, line 40 uses `$this->resource->id`, the "id" references a property of $this->resource not the local variable `$id`.